covid_model/db.py

```diff
@@ -20,58 +21,11 @@ def db_engine(db_name_env=None, db_user_env=None, db_host_env=None, db_credentia
     return create_engine(conn_string)
 
 
-def db_connect(db_name_env=None, db_user_env=None, db_host_env=None, db_credentials_env=None, env_prefix=''):
-    db_name = os.environ[db_name_env if db_name_env is not None else (env_prefix+'db_name')]
-    db_user = os.environ[db_user_env if db_user_env is not None else (env_prefix+'db_user')]
-    db_host = os.environ[db_host_env if db_host_env is not None else (env_prefix+'db_host')]
-    db_credentials = os.environ[db_credentials_env if db_credentials_env is not None else (env_prefix+'db_credentials')]
-
-    conn_string = "dbname='" + str(db_name) + "' user='" + str(db_user) + "' host='" + str(db_host) + "' password='" + str(db_credentials) + "'"
-
-    try:
-        conn = psycopg2.connect(str(conn_string))
-        conn.autocommit = True
-        return conn
-    except:
-        print("Unable to connect to the database")
-        raise
-
-
-def fetch_data(engine, sql=None, sql_path=None, save_to_file=None, force_rerun=True, manifest_path=None, update_manifest=True, **pd_read_args):
-    # get manifest file
-    manifest = None
-    this_manifest = None
-    if manifest_path:
-        manifest_file = open(manifest_path, 'r')
-        manifest = json.load(manifest_file) if os.stat(manifest_path).st_size != 0 else []
-        this_manifest_list = [m for m in manifest if m['csv_file'] == save_to_file]
-        this_manifest = this_manifest_list[0] if len(this_manifest_list) > 0 else None
-    # get data from csv, if appropriate
-    if save_to_file and not force_rerun and os.path.exists(save_to_file):
-        if this_manifest:
-            pd_read_args['parse_dates'] = [k for k, v in this_manifest['dtypes'].items() if v == 'datetime64[ns]']
-            pd_read_args['dtype'] = {k: v for k, v in this_manifest['dtypes'].items() if v != 'datetime64[ns]'}
-        data = pd.read_csv(save_to_file, **pd_read_args)
-    # otherwise get data from db
+def get_sqa_table(engine, schema, table):
+    if str(engine.engine.url).split('://')[0] == 'bigquery':
+        return Table(f'{schema}.{table}', MetaData(bind=engine), autoload=True)
```

Comment:
Looks like MetaData.bind is deprecated, may not be an issue if we don't upgrade sqlalchemy to v2.0
https://docs.sqlalchemy.org/en/14/core/metadata.html#sqlalchemy.schema.MetaData

Reply:
Ugh. This was a pain...for some reason metadata.reflect() wasn't finding any tables in BigQuery, so I switched to this. I'm honestly not 100% sure why this is working when the other approach wasn't. Might just leave this one be and live with the consequences down the line, unless you want to take a crack at it.